Helper/AutomatedTesting.php

```diff
@@ -213,8 +309,11 @@ public function getAutomatedTestingConfig()
                 ->setTax($tax)
                 ->setSubTotal($this->formatPrice($quote->getSubtotal()));
             $this->quoteRepository->delete($quote);
-
-            return $this->configFactory->create()->setStoreItems($storeItems)->setCart($cart);
+            $pastOrder = $this->convertToOrder($this->getPastOrder());
+            return $this->configFactory->create()
+                ->setStoreItems($storeItems)
+                ->setCart($cart)
+                ->setPastOrder($pastOrder);
```

Comment:
might as well just do
```suggestion

            return $this->configFactory->create()
                ->setStoreItems($storeItems)
                ->setCart($cart)
                ->setPastOrder($this->convertToOrder($this->getPastOrder()));
```